weldx/core.py

```diff
@@ -624,7 +671,7 @@ def units(self) -> str:
         Returns
         -------
         str:
-            Unit sting
+            Unit string
```

Comment:
This functions seems to return a `pint.Unit`? That is probably even better than working with strings

Reply:
Didn't check the function, just fixed a typo 😉. However, I think like with the `time` property, we should define what a `units` property is supposed to return and keep the interfaces aligned. A `pint.Unit` might be a more flexible option if it automatically converts to a string during a `print` or inside of a f-string. I think that most of the time the "user" calls this function he wants to print the unit in some way. The interfaces should be tailored to match the users' expectations.

Reply:
Updated the docstring to reflect the actual behavior